inputFiles/thermalSinglePhaseFlowFractures/egsCollab_thermalFlow/egsCollabLaunch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/tcsh
+##### These lines are for Slurm
+#SBATCH -N 1
```

Comment:
I suggest not committing this file, because it is very specific to Quartz (bank name, address of your folders, etc)

Reply:
honestly, all the egsCollab xmls should probably go somewhere else as well.